src/content/css/main.css

```diff
@@ -0,0 +1,273 @@
+.container {
+    position: relative;
+    min-height: 100vh;
+}
+
+.header {
+    position: absolute;
+    width: 100%;
+    /* height: 10%; */
+    height: 3.7em;
+    left: 0px;
+    top: 0px;
+    background: #282828;
+}
+
+.header button {
+    position: absolute;
+    width: 30%;
+    /* margin-left: 3%; */
+    left: 1%;
+    top: 0.25vh;
+    height: 95%;
+
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 50px;
+    line-height: 50%;
+    display: flex;
+    align-items: center;
+
+    background-color: Transparent;
+    background-repeat: no-repeat;
+    border: none;
+    cursor: pointer;
+    overflow: hidden;
+    outline: none;
+    color: #ffffff;
+    /* margin-top: 0.25vh; */
+}
+
+.search {
+    position: absolute;
+    width: 60%;
+    left: 33%;
+    top: 21%;
```

Comment:
Suggestion: Please avoid using such values. Stick to something like 0, 5, 10, 15, etc.